{
    "docs": "https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/shebang.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test shebang` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/shebang.html)] failed with 1 error:", 
            "output": "plugins/modules/netfoundry_endpoint.py:1:1: expected module shebang \"b'#!/usr/bin/python'\" but found: b'#!/usr/bin/env python3'"
        }
    ], 
    "verified": false
}
